src/main/java/com/github/manolo8/darkbot/core/manager/GroupManager.java

```diff
@@ -69,6 +71,9 @@ public void update() {
         inviteDict.update(API.readMemoryLong(groupAddress + 0x48));
 
         inviteDict.sync(invites, () -> new Invite(main.hero), invite -> invite.valid);
+
+        long groupMediatorAddress = main.facadeManager.groupMediator.address;
+        isBlockingInvites = API.readBoolean(API.readMemoryPtr(groupMediatorAddress, 0x30), 0xD4);
```

Comment:
```suggestion
        isBlockingInvites = API.readBoolean(groupMediatorAddress, 0x30, 0xD4);
```

Reply:
This will need an update to atom mask the nested loop calls